linux_os/guide/services/ssh/ssh_server/sshd_disable_gssapi_auth/kubernetes/shared.yml

```diff
@@ -0,0 +1,8 @@
+# platform = multi_platform_ocp,multi_platform_rhcos
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+{{{ kubernetes_sshd_set() }}}
+---
+{{{ kubernetes_sshd_dropin('GSSAPIAuthentication no', config_basename='75-complianceascode-sshd_disable_gssapi_auth.conf') }}}
```

Comment:
I wonder if we just want to make this change for all k8s remediations - https://github.com/ComplianceAsCode/content/blob/master/shared/templates/sshd_lineinfile/kubernetes.template#L11